File: SDKs/mcw.edu/1.1/go/pkg/models/operations/getmgimappingusingpost.go
package operations

import (
	"openapi/pkg/models/shared"
)

type GetMgiMappingUsingPostRequest struct {
	Request *shared.RgdidListRequest `request:"mediaType=application/json"`
}

type GetMgiMappingUsingPostResponse struct {
	Body        []byte
	ContentType string
	StatusCode  int64
}

File: SDKs/mcw.edu/1.1/python/src/sdk/models/operations/getmgimappingusingpost.py
from dataclasses import dataclass, field
from typing import Optional
from sdk.models import shared


@dataclass
class GetMgiMappingUsingPostRequest:
    request: Optional[shared.RgdidListRequest] = field(default=None, metadata={'request': { 'media_type': 'application/json' }})
    

@dataclass
class GetMgiMappingUsingPostResponse:
    content_type: str = field()
    status_code: int = field()
    body: Optional[bytes] = field(default=None)
    
